lib/db/db_sqlite.rb

```diff
@@ -0,0 +1,85 @@
+require 'db/db'
+require 'sqlite3'
+
+class DBSQLite < DB
+
+  def initialize(sqlite, table, columns)
```

Comment:
Why, as a client of `DBSQLite` in the ConMan application, do I need to know a table and it's columns?